apiserver/common/resource.go

```diff
@@ -129,10 +131,19 @@ func (rs *Resources) Count() int {
 	return len(rs.resources)
 }
 
+// FIXME(nvinuesa): This `StringResource` should be removed and they should not
+// be registered.
 // StringResource is just a regular 'string' that matches the Resource
 // interface.
 type StringResource string
 
+func (StringResource) Kill() {
+}
+
+func (StringResource) Wait() error {
+	return nil
+}
+
 func (StringResource) Stop() error {
```

Comment:
I know these are temporary, but technically we can remove the `Stop` methods here right?

Reply:
sure, removed